src/amperity/gocd/secret/vault/plugin.clj

```diff
@@ -23,22 +26,32 @@
 ;; ## Plugin Initialization
 
 (defn initialize!
-  "Initialize the plugin state."
+  ;; should really just be named initialize, not initialize!, all mutation is self contained
```

Comment:
🤷‍♂ it does change the root binding of the logger and will make external network IO to authenticate the client, so the bang feels warranted to me.